src/UI/wwwroot/V4/Identity/lib/jquery-validation-unobtrusive/jquery.validate.unobtrusive.js

```diff
@@ -1,16 +1,17 @@
 // Unobtrusive validation support library for jQuery and jQuery Validate
-// Copyright (C) Microsoft Corporation. All rights reserved.
-// @version v3.2.9
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+// @version v3.2.11
 
 /*jslint white: true, browser: true, onevar: true, undef: true, nomen: true, eqeqeq: true, plusplus: true, bitwise: true, regexp: true, newcap: true, immed: true, strict: false */
 /*global document: false, jQuery: false */
 
 (function (factory) {
     if (typeof define === 'function' && define.amd) {
         // AMD. Register as an anonymous module.
-        define("jquery.validate.unobtrusive", ['jquery.validation'], factory);
+        define("jquery.validate.unobtrusive", ['jquery-validation'], factory);
```

Comment:
Is this intentional? This is changing the dependency on the define

Reply:
https://github.com/aspnet/jquery-validation-unobtrusive/issues/79. Intentional, this copy was just more than one version out of date.